Makefile

```diff
@@ -140,6 +140,7 @@ build-all-images: $(addsuffix -build-image,$(IMAGES))
 		--build-arg LDFLAGS='$(LDFLAGS)' \
 		.
 	docker tag $(DOCKER_IMAGE_PREFIX)$*:$(IMMUTABLE_DOCKER_TAG) $(DOCKER_IMAGE_PREFIX)$*:$(MUTABLE_DOCKER_TAG)
+	kind load docker-image $(DOCKER_IMAGE_PREFIX)$*:$(IMMUTABLE_DOCKER_TAG) || echo >&2 "kind not installed or error loading image"
```

Comment:
I'd definitely prefer a separate set of targets for loading images into a kind cluster.  How about `load-all-images` and `%-load-image` for each?  I was playing around a little and came up with this:
```
.PHONY: load-all-images
load-all-images: $(addsuffix -load-image,$(IMAGES))

%-load-image:
	@echo "Loading $(DOCKER_IMAGE_PREFIX)$*:$(IMMUTABLE_DOCKER_TAG)" && \
		KUBECONFIG="$(shell kind get kubeconfig-path --name="kind")" \
			kind load docker-image $(DOCKER_IMAGE_PREFIX)$*:$(IMMUTABLE_DOCKER_TAG) \
				|| echo >&2 "kind not installed or error loading image: $(DOCKER_IMAGE_PREFIX)$*:$(IMMUTABLE_DOCKER_TAG)"
```

There I added an informative echo, automatically switching kube context to kind (that way, I can load images to my kind cluster even if my current context is on a different one.  Maybe this is overkill/not needed... or perhaps it isn't Windows compatible?  Feel free to ignore, if so), and then run the `kind load` command.

(The `@` symbol prefixing the command string just silences printing out the actual command string when running)

Either way, if this approach sounds good, we can update the docs, e.g, `... make build-all-images load-all-images`

Reply:
Great addition, thanks! I copied this to the Makefile, works like a charm. Regarding the KUBECONFIG set on `%-load-image`, not sure if it's needed, since you should set the KUBECONFIG before calling `load-all-images`, provided that you follow the instructions, of course.